getting errors with using the stylex CLI in next 15 App dir.

### ISSUE
 - Im getting the following error while using the stylex CLI with next 15 while following the example in https://github.com/facebook/stylex/pull/618/commits/eab671c6e7f925ae2eac8f08cd531c79a6d38eae:
 `'stylex.create' should never be called at runtime. It should be compiled away by '@stylexjs/babel-plugin'`


### CODES

.stylex.json5:
```js
{
  input: ['./source'],
  output: ['./src'],
  cssBundleName: 'stylex_bundle.css',
  useCSSLayers: true,
  babelPresets: [
    ['@babel/preset-typescript', { allExtensions: true, isTSX: true }],
    // '@babel/preset-react',
  ],
  // modules_EXPERIMENTAL: [
  //   ['@stylexjs/open-props', { ignore: ['src', '__tests__'] }],
  // ],
  // watch: true,
}
```

next.config.ts (default nothing changed.):
```tsx
import type { NextConfig } from "next";

const nextConfig: NextConfig = {
  /* config options here */
};

export default nextConfig;
```



and here's my **package.json** incase its needed:
```json
{
  "name": "next_cli_stylx_three_appdir",
  "version": "0.1.0",
  "private": true,
  "scripts": {
    "predev": "stylex --config ./.stylex.json5",
    "dev": "stylex --config ./.stylex.json5 -w & next dev",
    "prestart": "stylex --config ./.stylex.json5 && next build",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "@babel/preset-typescript": "^7.26.0",
    "@babel/runtime": "^7.26.0",
    "@stylexjs/babel-plugin": "^0.9.3",
    "@stylexjs/cli": "^0.9.3",
    "@stylexjs/stylex": "^0.9.3",
    "next": "15.0.3",
    "react": "19.0.0-rc-66855b96-20241106",
    "react-dom": "19.0.0-rc-66855b96-20241106"
  },
  "devDependencies": {
    "@types/node": "^20",
    "@types/react": "^18",
    "@types/react-dom": "^18",
    "typescript": "^5"
  }
}
```

and here's the code of me trying to use the stylex.create() in app/page.tsx
```tsx
import "../stylex_bundle.css";
import * as stylex from "@stylexjs/stylex"


export default function Home() {
  return (
    <>
      <div {...stylex.props(styles.rootDiv)}> 

      </div>
    </>
  );
}

const styles = stylex.create({
  rootDiv: {width: "100%", minHeight: "100dvh", background: "green"}
})
```

_**I have made all the changes relating to the src also.**_
If needed, ill put the project up on github.

thanks for all this help. Im having an issue of when i try to run npm run dev now, its generating a new src folder along with layout.js page.js etc. which is giving me this error:

### ERROR
`Module not found: Can't resolve '..\stylex_bundle.css'
Did you mean './..\stylex_bundle.css'?
Requests that should resolve in the current directory need to start with './'.
Requests that start with a name are treated as module requests and resolve within module directories (node_modules).
If changing the source code is not an option there is also a resolve options called 'preferRelative' which tries to resolve these kind of requests in the current directory too.
> 1 | import "..\\stylex_bundle.css";
    | ^
  2 | import * as stylex from "@stylexjs/stylex";
  3 | import "./globals.css";
  4 | export const metadata = {

https://nextjs.org/docs/messages/module-not-found`

### Codes being generated from the npm run dev command:

**layout.js**
`import "..\\stylex_bundle.css";
import * as stylex from "@stylexjs/stylex";
import "./globals.css";
export const metadata = {
  title: "Create Next App",
  description: "Generated by create next app"
};
export default function RootLayout({
  children
}) {
  return <html {...{
    className: "x1ghz6dp x1717udv"
  }} lang="en">
      <body {...{
      className: "x1ghz6dp x1717udv xg6iff7 x100uywg"
    }}>{children}</body>
    </html>;
}`
**page.js:** 
`import "..\\stylex_bundle.css";
import { props, create, keyframes } from "@stylexjs/stylex";
// Although StyleX can compile `@/` paths, TS can only resolve
// them to one location. This will be fixed in the CLI soon.
import { fonts, tokens } from "../vars.stylex";
export default function Home() {
  return <main {...{
    className: "xg6iff7 x100uywg x78zum5 xdt5ytf x6s0dn4 xl56j7k"
  }}>
      <h1 {...{
      className: "xer8iyq x8cw2a4 xo5v014 x1ghz6dp"
    }}>Welcome to Next.js & StyleX!</h1>
      <p {...{
      className: "xer8iyq xvewgow x1evy7pa xman319"
    }}>
        Get started by editing <code>pages/index.tsx</code>
      </p>
      <div {...{
      className: "xpyat2d x1oysuqx xybe50t xtursrm x1ke1kyy x18ebdri xa4qsjk"
    }} />
    </main>;
}
const gradientColors = ["#ffadad", "#ffd6a5", "#fdffb6", "#caffbf", "#9bf6ff", "#a0c4ff", "#bdb2ff", "#ffc6ff", "#ffadad"].join(", ");
const rotate = "xa503dy-B";`

_along with stylex_bundle.css which has alot of css classes with styles aswell as vars.stylex.js. I don't think those file contents have anything to do with the error so i wont share it but if neccessary I will._  

Is there any way to stop generating a new src file when i run npm run dev?

I tried doing that, it's giving me this error.

```
Module not found: Can't resolve '..\stylex_bundle.css'
Did you mean './..\stylex_bundle.css'?
Requests that should resolve in the current directory need to start with './'.
Requests that start with a name are treated as module requests and resolve within module directories (node_modules).
If changing the source code is not an option there is also a resolve options called 'preferRelative' which tries to resolve these kind of requests in the current directory too.
> 1 | import "..\\stylex_bundle.css";
    | ^
  2 | import { props, create, keyframes } from "@stylexjs/stylex";
  3 | // Although StyleX can compile `@/` paths, TS can only resolve
  4 | // them to one location. This will be fixed in the CLI soon.

https://nextjs.org/docs/messages/module-not-found
```

Also how can i stop it from generating a new src folder everytime with a layout.js and page.js since I already have a layout.tsx and page.tsx ?

--- a/packages/cli/src/files.js
+++ b/packages/cli/src/files.js
@@ -72,7 +72,7 @@ export function isJSFile(filePath: string): boolean {
 
 // e.g. ./pages/home/index.js -> ../../stylex_bundle.css
 export function getRelativePath(from: string, to: string): string {
-  const relativePath = path.relative(path.parse(from).dir, to);
+  const relativePath = path.posix.relative(path.parse(from).dir, to);
   return formatRelativePath(relativePath);
 }
 

